Package.swift

```diff
@@ -32,7 +32,8 @@ let package = Package(
         .library(name: "MessageKit", targets: ["MessageKit"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/nathantannar4/InputBarAccessoryView", .upToNextMajor(from: "5.5.0"))
+//        .package(url: "https://github.com/nathantannar4/InputBarAccessoryView", .upToNextMajor(from: "5.5.0"))
+        .package(url: "https://github.com/nathantannar4/InputBarAccessoryView", .branch("master"))
```

Comment:
Should cut release of IBAV before merging this probably? And drop iOS 12 too there

Reply:
Sounds good 👍

Reply:
We can now point to 6.0.0 release